CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+7.2.0
+-----
+* Add application_name configuration option
+* Update login page copy
+
```

Comment:
this can probably be a minor bump. Also we haven't actually release 7.1 yet if you to sneak it in for that release